tests/fail-dep/libc/eventfd_block_read_twice.rs

```diff
@@ -14,7 +14,7 @@ use std::thread;
 // 2. Thread 2 blocks.
 // 3. Thread 3 unblocks both thread 1 and thread 2.
 // 4. Thread 1 reads.
-// 5. Thread 2's `read` deadlocked.
+// 5. Thread 2's `read` can neber complete -> deadlocked.
```

Comment:
nit: you mean never complete, not neber complete?

Reply:
I meant "never" :joy: